x/epochs/spec/01_concepts.md

```diff
@@ -0,0 +1,9 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+The `epochs` module defines on-chain timers that execute at fixed time intervals. Other Evmos modules can then register logic to be executed at the timer ticks. We refer to the period in between two timer ticks as an "epoch".
```

Comment:
@AustinoBombino I like the EVMOS documentation structure I was thinking of using the same tools to generate the protocol and broader documentation of Elys based on those spec files.

here maybe a minor change to rename the occurrences of `Evmos` to `Elys` thanks